docs/UserGuide.md

```diff
@@ -832,11 +832,27 @@ Exits the program.
 
 Format: `exit`
 
-### 4.11 Saving the data
+### 4.11 Saving the Data
 
 Fine$$e data is saved in the hard disk automatically after any command that changes the data.
 There is no need to save manually.
 
+### 4.12 Command History
+
+Accidentally entered the wrong command and wish to modify it without typing it out again fully?
+Simply press the ↑ or ↓ arrow keys on your keyboard to navigate through your command history.
+
+* The command history keeps track of the latest 50 commands entered.
+* The command input box must be focused on.
```

Comment:
If anyone can find a more CELC-friendly way of phrasing this without introducing any ambiguities, I'm all ears.